GameData/000_FilterExtensions Configs/Default/Mod_SMM.cfg

```diff
@@ -0,0 +1,22 @@
+CATEGORY
```

Comment:
That one was a mistake on my part - I thought I removed that config was still a WIP